h2o-core/src/main/java/water/fvec/persist/FramePersist.java

```diff
@@ -0,0 +1,111 @@
+package water.fvec.persist;
+
+import water.*;
+import water.fvec.Chunk;
+import water.fvec.Frame;
+import water.fvec.Vec;
+import water.util.FileUtils;
+
+import java.net.URI;
+
+import static water.fvec.persist.PersistUtils.*;
+
+public class FramePersist {
+    
+    static {
+        TypeMap.onIce(FrameMeta.class.getName());
+    }
+
+    private final Frame frame;
+
+    public FramePersist(Frame frame) {
+        this.frame = frame;
+    }
+    
+    private static class FrameMeta extends Iced<FrameMeta> {
+        Key<Frame> key;
+        String[] names;
+        Key<Vec>[] keys;
+        Vec[] vecs;
```

Comment:
How does this save Vec.ESPC?

Reply:
good catch, just found it by my self, pushed fix